_docs/how_to_run_acceptance_tests.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: How to run beaker acceptance tests
+date: 2025-06-17
+summary: A very short description of how to run the vox pupuli acceptance test suite for puppet modules.
+---
+
+- [Installing System Dependencies](#installing-system-dependencies)
+  - [Runtime Environment](#runtime-environment)
+  - [Configuring Bundler](#configuring-bundler)
+- [Setting up to run a test suite](#setting-up-to-run-a-test-suite)
+  - [Listing platforms](#listing-platforms)
+  - [Running Beaker](#running-beaker)
+- [Adding data to a run](#adding-data-to-a-run)
+- [Resources](#resources)
+
+# Running Vox Pupuli Acceptance Tests
+
+The testing and development tools have a bunch of dependencies, all managed by [bundler](http://bundler.io/).
+The tests use the latest version of Puppet.
+
+## Installing system dependencies
+
+### Runtime Environment
+
+Beaker uses a local hypervisor or container stack to build required OS/Distributions to test.
+By default this is docker, but vagrant via VirtualBox and libvirt as well as VMWare vSphere are available.
+
+In general, docker is preferred for modules, and is what the Vox Pupuli CI systems use by default, making it the best option.
```

Comment:
It's not the best: it's a container and you can't test everything. SELinux can't be used in it for example. I'd say it's the easiest to get running though, making it the preferred option in many cases.